openedx/core/djangoapps/user_authn/views/password_reset.py

```diff
@@ -516,7 +519,11 @@ def post(self, request, *args, **kwargs):
         if password_reset_successful and is_account_recovery:
             self._handle_password_creation(request, updated_user)
 
-        send_password_reset_success_email(updated_user, request)
+        if is_testing_environment():
+            send_password_reset_success_email(updated_user, request)
+        else:
+            compose_and_send_adg_password_reset_success_email(updated_user)
```

Comment:
Lets please follow the way used in this method `adg.lms.student.helpers.send_account_activation_email`.
Create new method in adg directory and use it in place of `send_password_reset_success_email ` and that method will call `send_password_reset_success_email ` if is_testing_env else use our adg logic. This way we will have minimum core changes.